core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -96,8 +95,9 @@ enum class UniqueType(val text: String, vararg targets: UniqueTarget, val flags:
     // Stat percentage boosts
     StatPercentBonus("[relativeAmount]% [stat]", UniqueTarget.Global, UniqueTarget.FollowerBelief),
     StatPercentBonusCities("[relativeAmount]% [stat] [cityFilter]", UniqueTarget.Global, UniqueTarget.FollowerBelief),
-    StatPercentFromObject("[relativeAmount]% [stat] from every [tileFilter/specialist/buildingName]", UniqueTarget.Global, UniqueTarget.FollowerBelief),
-    AllStatsPercentFromObject("[relativeAmount]% Yield from every [tileFilter]", UniqueTarget.FollowerBelief, UniqueTarget.Global),
+    // Todo: Add support for specialist next to buildingName
```

Comment:
I understood that like:
> .... from every [tileFilter/buildingFilter/specialistFilter]", .....

Reply:
Basically what JackRainy said. `StatsFromObject` has support for `tileFilter`,`specialist` and `buildingFilter`, so it would make sense to also have support for all three of these for `StatPercentFromObject`.